tools/shared/utils/global.h

```diff
@@ -159,6 +159,8 @@ typedef struct {
   UINT xon;
   LOGICAL ucase;
   char **idir;
+  char **linker_directives;
+  char *llvm_target_triple;
```

Comment:
I was working on adding `llvm_target_triple`, so I combined that work. I can remove this and send a new PR. Let me know